crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -123,6 +126,17 @@ impl Name {
             })
             .transpose()?;
 
+        if let Some(case) = auto_case.cxx {
+            if cxx_name.is_none() {
+                cxx_name = Some(ident.to_string().to_case(case));
+            }
+        }
+        if let Some(case) = auto_case.rust {
+            if rust_name.is_none() {
+                rust_name = Some(format_ident!("{}", ident.to_string().to_case(case)));
+            }
+        }
```

Comment:
Do we only auto case if there is no cxx name *and* no rust name in all cases?

Or does it apply when only the name we are affecting is missing ? :thinking:

Reply:
Spoke with Leon, lets go with only when both are none for now

Reply:
Done